packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -47,15 +51,24 @@ const CardContext = React.createContext<{
  *
  * @public
  */
-export const Card: CardInterface = ({ disabled, href, onClick, children, isSelected, className, ...htmlProps }) => {
+export const Card: CardInterface = ({
+  disabled,
+  href,
+  onClick,
+  children,
+  isSelected,
+  isCompact,
+  className,
```

Comment:
This `classname` is there due to `CardConteiner`, isn't it? So it's deprecated as that component, no?

Reply:
I guess so, I didnt make changes to that, just added another prop 🤷